### PR TITLE
Changed Share modal styles

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Changes styles of share file modal.

## How does it look?

Screen Shot 2022-05-02 at 19 25 35

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Review by oscarocastellanos — COMMENTED** [review]
LGTM

File: portal-ui/src/icons/LinkIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (prakashsvmx) [inline portal-ui/src/icons/LinkIcon.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

**Review by prakashsvmx — APPROVED** [review]
LGTM :+1:

File: portal-ui/src/icons/LinkIcon.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/icons/LinkIcon.tsx:2]: Will fix in an upcoming PR

**Pull request merged**